packages/jest-enzyme/README.md

```diff
@@ -50,3 +50,9 @@ If you are using Create React App, instead of adding to your `package.json` as a
  ```
 
  This is documented on Create React App at the bottom of the [Testing Components](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#testing-components) section. There is also more information about [Initializing Test Environment](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#initializing-test-environment).
+ 
+ Also, instead of installing the latest version you should stick with versions below 3, because Create React App currently [uses](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/package.json#L50) a version of jest which will cause [this issue](https://github.com/blainekasten/enzyme-matchers/issues/71):
```

Comment:
That is a good wording, but react create app currently uses 18.1.0. Until they upgrade to 19.0.0, we can't put a number on X. 

So I guess the proposed wording is accurate right now, pending rewording when RCA is upgrading their dependency. I have subscribed to the [relevant issue](https://github.com/facebookincubator/create-react-app/issues/1604) in the RCA repo, so I'll be able to know when to create a new PR.